src/osha/oira/services/survey_publication_count.py

```diff
@@ -0,0 +1,16 @@
+from plone import api
+from plone.restapi.services import Service
+
+
+class SurveyPublicationCountService(Service):
+    """A JSON service version of @@survey-publication-count"""
+
+    def reply(self):
+        view = api.content.get_view(
+            "survey-publication-count", self.context, self.request
+        )
+        view()
```

Comment:
I checked https://github.com/euphorie/osha.oira/pull/338/files and made some suggestions on that one.
If you follow them, you will not need to do this view call to initialize the rows attribute.